dashboard/app/public_json_api.go

```diff
@@ -53,9 +59,11 @@ func getExtAPIDescrForBugPage(bugPage *uiBugPage) *publicAPIBugDescription {
 			var res []vcsCommit
 			for _, commit := range bugPage.Bug.Commits {
 				res = append(res, vcsCommit{
-					Title: commit.Title,
-					Hash:  commit.Hash,
-					Link:  commit.Link,
+					Title:  commit.Title,
+					Hash:   commit.Hash,
+					Link:   commit.Link,
+					Repo:   repoName,
+					Branch: branchName,
```

Comment:
First crash repo does not necessary fix commit repo.

Reply:
I'll exclude it here and extend CommitInfo collection in the next PR.